server/cmd/mmctl/commands/compliance_export.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/client"
+	"github.com/spf13/cobra"
+)
+
+var ComplianceExportCmd = &cobra.Command{
+	Use:   "compliance_export",
```

Comment:
Could this use dash instead of underscore? I'm trying to cleanup underscores from the commands.
```suggestion
	Use:   "compliance-export",
```

Actually, even better (for future ease of expansion) maybe we can go with:

```suggestion
	Use:   "compliance export",
```

Maybe we will want to add another compliance feature.